front/cypress/e2e/events/events_page.cy.ts

```diff
@@ -54,6 +54,11 @@ describe('All events page', () => {
 
     // Select filters
     cy.get('.e2e-project-filter-selector').should('exist');
+
+    // is accessible
```

Comment:
This is how we add accessibility checks with the new setup.

Reply:
Super cool! Will make sure to use this for the survey changes :+1: